### PR TITLE
SKU pack information

### PR DESCRIPTION
**Pull request by zyoung51**

### CODE REVIEW

File: docs/rackhd/skus.rst [hunk 1]
```diff
@@ -215,3 +260,42 @@ SKUs are defined via JSON, with these required fields:
...
+The SKU pack requires the 'config.json' to be at the root of the tar.gz file.  A typical
+package may have a static and template directory.
+
+.. code-block:: shell
+
+    tar tzf pack.tar.gz:
+    config.json
+    static/
+    static/common/
+    static/common/discovery.overlay.cpio.gz
```
Comment (benbp) [inline docs/rackhd/skus.rst]: This just defaults to http (`/static/http/common`)? So tftp file overrides aren't supported currently?

File: docs/rackhd/skus.rst [hunk 2]
```diff
@@ -215,3 +260,42 @@ SKUs are defined via JSON, with these required fields:
...
+        {
+          "path": "dmi.dmi.base_board.manufacturer",
+          "contains": "Intel"
+        },
+        {
+          "path": "dmi.memory.total",
+          "equals": "32946864kB"
+        }
+      ],
+      "httpStaticRoot": "static",
```
Comment (benbp) [inline docs/rackhd/skus.rst]: Oh, I guess this kind of answers my question above. You point to it.

**Comment by benbp** [comment]
:+1:

**Comment by heckj** [comment]
:+1: Looks great, thank you!

**Comment by jlongever** [comment]
+1

**Pull request merged**